### DISCUSSION TITLE
Link to salib for sensitivity analysis

### DISCUSSION THREAD

**Discussion by MichaelClerx** [pints-team/pints#1615] [Ideas/Science]
SA seems like an obvious third main function for Pints 

https://salib.readthedocs.io/en/latest/

**Comment by chonlei** [comment]
Do we have any algorithm that actually uses global sensitivity?! I think all are local sensitivity _only_.

Why do we want _global_ sensitivity in Pints?

**Comment by MichaelClerx** [comment]
Not for the algorithms, just because it's useful to do the analysis!
